charts/ark-pentaho-ce/files/repository.spring.properties

```diff
@@ -0,0 +1,40 @@
+#
+# *******************************************************************************
+#
+# Copyright (C) 2002-2017 by Hitachi Vantara : http://www.pentaho.com
+#
+# *******************************************************************************
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#Tue Mar 26 17:50:44 EDT 2013
+singleTenantAdminDefaultUserName=admin

```

Comment:
Please use only LF for line ending (UNIX style), rather than CRLF (Windows style). Ditto for the rest of the file.

Reply:
thanks for spotting this out changed to LF.